sdk/src/test/scala/io/horizen/account/state/ForgerStakeV2MsgProcessorTest.scala

```diff
@@ -489,6 +497,220 @@ class ForgerStakeV2MsgProcessorTest
     Hash.ZERO
   )
 
+
+  @Test
+  def testRegisterForger(): Unit = {
```

Comment:
Could you please add a test on the register event and on the balances?

Reply:
Added